app/views/contexts/section_preview_context.py

```diff
@@ -0,0 +1,97 @@
+from functools import cached_property
+from typing import Any, Mapping, Optional, Union
+
+from werkzeug.datastructures import ImmutableDict
+
+from app.data_models import AnswerStore, ListStore, ProgressStore, QuestionnaireStore
+from app.data_models.metadata_proxy import MetadataProxy
+from app.questionnaire import QuestionnaireSchema
+from app.questionnaire.location import Location
+from app.utilities import safe_content
+
+from .context import Context
+from .preview import PreviewGroup
+
+
+class SectionPreviewContext(Context):
+    def __init__(
+        self,
+        language: str,
+        schema: QuestionnaireSchema,
+        answer_store: AnswerStore,
+        list_store: ListStore,
+        progress_store: ProgressStore,
+        metadata: Optional[MetadataProxy],
+        response_metadata: Mapping,
+        current_location: Location,
+        questionnaire_store: QuestionnaireStore,
+    ):
+        super().__init__(
+            language,
+            schema,
+            answer_store,
+            list_store,
+            progress_store,
+            metadata,
+            response_metadata,
+        )
+        self.current_location = current_location
+        self.questionnaire_store = questionnaire_store
+        self.language = language
+
+    def __call__(
+        self, return_to: Optional[str] = "section-summary"
```

Comment:
Preview has not context of `return_to` therefore should be removed.

Reply:
Removed.